### PR TITLE
Storage length

### PR DESCRIPTION
**Pull request by Rcshort123**
### Description

Created the beginning of the length.md by creating the path folders to the file. I have done an introduction of the method and the syntax for string and array. As this is the first contribution I wanted to make a pull request to double-check that I am going about this the right way before I get too far into it. plus small requests are better from what I learned on Codecademy so far. thank you

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x ] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/storage/terms/length.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# .length()
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/storage/terms/length.md:1]:
```suggestion
```
Add the table here like this:
image
```
---
Title: 'Selection-Sort Algorithm'
Description: 'Returns a sorted array'
Subjects:
  - 'Computer Science'
Tags:
  - 'Algorithms'
  - 'Arithmetic'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---
```
You have to change the names and description according to the entry

File: content/javascript/concepts/storage/terms/length.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+.length() is used on objects such as strings and arrays to return the length of the object.
+The .length() function does vary when used on string vs arrays. 
+
+## Syntax
+length is called on a string through the following syntax and will output the number of characters of the string. 
+```
+const pet = dog;
+console.log(pet.length);
+///returns 3;
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/storage/terms/length.md:11]:
```suggestion
```

File: content/javascript/concepts/storage/terms/length.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# .length()
+
+.length() is used on objects such as strings and arrays to return the length of the object.
+The .length() function does vary when used on string vs arrays. 
+
+## Syntax
+length is called on a string through the following syntax and will output the number of characters of the string.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/storage/terms/length.md:7]:
```suggestion
length is called on a string through the following syntax and will output the number of characters of the string. 

```

File: content/javascript/concepts/storage/terms/length.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# .length()
+
+.length() is used on objects such as strings and arrays to return the length of the object.
+The .length() function does vary when used on string vs arrays. 
+
+## Syntax
+length is called on a string through the following syntax and will output the number of characters of the string. 
+```
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/storage/terms/length.md:8]:
```suggestion
```shell
```

File: content/javascript/concepts/storage/terms/length.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+const pet = dog;
+console.log(pet.length);
+///returns 3;
+```
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/storage/terms/length.md:12]:
```suggestion
```

```

File: content/javascript/concepts/storage/terms/length.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+## Syntax 
+
+length is called on an array through the following syntax.
+
+```
+const array = [1, 2, 3, 4];
+console.log(array.length);
+/// returns 4
+```
+Remember arrays are 0 Indexed but .length will count the 0 as 1 in it's count.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/storage/terms/length.md:22]: Why not remove all this and add this code block in the above syntax itself. After that create a example section for the code blocks

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have some comments for you

**Comment by Rcshort123** [comment]
thank you @dakshdeepHERE for the comments I'll get working on them and make another request when I have it more fleshed out. =)

**Comment by dakshdeepHERE** [comment]
> thank you @dakshdeepHERE for the comments I'll get working on them and make another request when I have it more fleshed out. =)

Sure good luck with the changes.

**Comment by SSwiniarski** [comment]
> thank you @dakshdeepHERE for the comments I'll get working on them and make another request when I have it more fleshed out. =)

@Rcshort123 You don't need to make another PR, we can see all the changes in this PR. Also yo need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=2705) both as @Rcshort123 and @shortryan91. Also, @shortryan91 will be the GitHub user that will recieve credit for this entry, since that's the user that submitted the PR.

**Comment by SSwiniarski** [comment]
@Rcshort123 looking at this we have two problems:

1. You have no metadata at the start of your entry. You can learn how to add the metadata from the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).
2.  Second, this entry is not about the correct property. The array and string `length` properties have their own entries. What we want here is the `Storage.length` property. That is, the `length` property of the `Storage` object. You need to rewrite this to cover that property.

Tag me when you've addressed these issues and I'll do a first maintainer review.

**Comment by SSwiniarski** [comment]
@Rcshort123 are you going to submit another PR or do you want to be unassigned from this issue?

**Comment by Rcshort123** [comment]
@SSwiniarski Yes I will submit another request, as you mentioned I pull requested with the wrong account, and I would like to have it credited to the right account, and its good practice for me to make sure. I'll be sure to @ you in the new request thanks for all the help so far =)

**Pull request closed without merging**